### ISSUE TITLE
Is this plugin dead?

### ISSUE THREAD

**Issue by FibreFoX** [FibreFoX/javafx-gradle-plugin#119]
**TL;DR** no, its not dead, will be replaced in the near future.

This is some notifaction-issue, so please read this before posting.

No, this plugin is not (yet) dead, but the future got very fragile. This plugin started as some personal experiment for getting more knowledge of Gradle, which I never touched using it in production: Before getting into creating this plugin, I was put into charge of the [javafx-maven-plugin](https://github.com/javafx-maven-plugin/javafx-maven-plugin) which got me a lot of knowledge about github, maven and the whole javafx-world, which I too never did in production. This was purely for personal interests.

As the plugin seemed to get popular (and I hope it really helped a lot of people), the issues raised and made a lot of influence, but never helped to avoid being closely dependent to the installed JDK, which opened a lot of debugging-sessions and source-code-readings.

The javapackager is not very well designed, but the main purpose still is a good one at it's core. Sadly this made me take a lot of problems into this plugin. The override-feature, which is some good idea at some points, is not fully implemented by the javapackager-bundlers, making it impossible to have some additional files at some places, especially for mac-users, but I tried my best to workaround of a lot, even not always being the best way.

This plugin **will not have** a lot of shiny future, because **it'll get superseded** by something else I'm actually working on nearly every day. A lot of old design-decisions are getting cut and thrown away, making it even more useful for all of you developers.

Do not ask for any ETA, please live with the plugin in it's current state, as long as you can.

**Comment by swpalmer** [comment]
Is your new work based on jpackage tool to be included with Java 12?  https://openjdk.java.net/jeps/343

**Comment by Alexilmarranen** [comment]
It seems that the main problem of support in the new JDK of this plugin can be solved in the future. https://openjdk.java.net/jeps/343
Do you have plans to support this new packager?

**Comment by FibreFoX** [comment]
I'm back to deep dive into the topic ;)
